### PR TITLE
 Fix about us page

### PR DESCRIPTION
**Pull request by Lukshaolya**
fix section on about us page

⚠️ Please check all items below before review. ⚠️

- [x] Status in Jira
- [x] Code formatting and linting
- [x] No console logs or debug statements
- [x] No unnecessary comments
- [ ] Performance considerations
- [ ] Cross-browser and device testing
- [ ] Changes are tested locally
- [ ] Unit tests and test coverage

### CODE REVIEW

**Review by hapikus — APPROVED** [review]

File: src/pages/Info/aboutProject/AboutProgect.tsx
Comment (SwallowOnes) [inline src/pages/Info/aboutProject/AboutProgect.tsx:1]: AboutProdject good practice to name file just like modules with same name, TSX components with lower case

File: src/pages/Info/aboutProject/aboutProject.module.css
Comment (SwallowOnes) [inline src/pages/Info/aboutProject/aboutProject.module.css:1]: this file names right

File: src/pages/Info/aboutProject/AboutProgect.tsx [hunk 1]
```diff
@@ -521,12 +516,10 @@ function AboutProject() {
                         </div>
                         <div className={styles.timePeroidTextLeft}>
                           <p>
-                            Login page. Conducted discussions on the possibility
-                            of using MOBX instead of REDUX. Code reviews and
-                            requested implementations. discussed topics such as
-                            the vite.config.ts file, implementation and
-                            validation of dobValid.ts, and the database for
-                            products.
+                            Login page.Code reviews and requested
```
Comment (SwallowOnes) [inline src/pages/Info/aboutProject/AboutProgect.tsx]: .Code wthout space

File: src/pages/Info/aboutProject/AboutProgect.tsx [hunk 2]
```diff
@@ -521,12 +516,10 @@ function AboutProject() {
...
+                            implementations. discussed topics such as the
```
Comment (SwallowOnes) [inline src/pages/Info/aboutProject/AboutProgect.tsx]: Discussed with uppercase

**Review by SwallowOnes — COMMENTED** [review]
Some notice to change

**Review by hapikus — APPROVED** [review]

**Pull request merged**